client/gutenberg/extensions/editor-notes/index.js

```diff
@@ -5,32 +5,27 @@
  */
 import { registerBlockType } from '@wordpress/blocks';
 import { RichText } from '@wordpress/editor';
+import Card from 'components/card';
+import Ribbon from 'components/ribbon';
 
-import './style.scss';
+import styles from './style.scss';
 
 const attributes = {
 	notes: {
 		type: 'array',
 	},
 };
 
-const edit = ( { attributes: { notes }, className, isSelected, setAttributes } ) => (
-	<div className={ isSelected ? 'is-selected' : '' }>
-		{ ! isSelected && (
-			<span className="editor-notes__editor-indicator">
-				<span role="img" aria-label="notebook">
-					📔
-				</span>
-				Editor's Notes: hidden from rendered page
-			</span>
-		) }
+const edit = ( { attributes: { notes }, className, setAttributes } ) => (
+	<Card highlight="error" className={ `${ className } ${ styles.box }` }>
```

Comment:
Here's an interesting case. I'm curious how this will behave, if it will be deterministic and predictable, and what we'd need to do to ensure that it is.

We have a `Card`, with its [low-specificity styles](https://github.com/Automattic/wp-calypso/pull/26933/files#diff-7675d4e8d29c184b4dc0c9f255cb14e7R7): `background: $white`.

This component [appears to provide an override](https://github.com/Automattic/wp-calypso/pull/26933/files#diff-0bd95e492876a9313ca9cecaa155a2bfR2): `background-color: lighten( $alert-red, 40% )`.

Based on your screenshot, this appears to apply the red background as desired. Does some sort of spec describe how this works? Given that both of these selectors are simple `.classes`, they have the same specificity and depend on the cascade (last one wins). Does [`import Card`](https://github.com/Automattic/wp-calypso/pull/26933/files#diff-77b736110f622d5ec97bc00d59939031R8) ensure that Card's styles are loaded first, and this component's styles loaded after?

```css
/**
* Card styles loads first, good 👍
* This apparently is what happens.
*/
.card {
 	background: $white;
}
.box {
	background-color: lighten( $alert-red, 40% );
}

/**
* Editor notes styles loads first, bad 👎
* This doesn't appear to be the case.
*/
.box {
	background-color: lighten( $alert-red, 40% );
}
.card {
 	background: $white;
}
```

🤔 Is this the way we'd want to work with css-modules?